commons/src/main/java/fr/free/nrw/commons/category/CategorizationFragment.java

```diff
@@ -87,113 +95,72 @@ public void writeToParcel(Parcel parcel, int flags) {
         }
     }
 
-    private class CategoriesUpdater extends AsyncTask<Void, Void, ArrayList<String>> {
+    protected ArrayList<String> recentCatQuery() {
+        ArrayList<String> items = new ArrayList<String>();
```

Comment:
Changing it to a List makes it return a List, which makes it incompatible with the preexisting AyncTask result types etc.